virtcontainers/pkg/oci/utils_test.go

```diff
@@ -815,7 +816,12 @@ func TestGetShmSizeBindMounted(t *testing.T) {
 	err = os.Mkdir(shmPath, 0700)
 	assert.Nil(t, err)
 
-	size := 8192
+	size := 0
```

Comment:
I think the following code is simpler and easy to read, could you add a comment explaining why `65536`?  alignment? page size?

```go
	size := 8192
	if runtime.GOARCH == "ppc64le" {
		size = 65536
	}
```

Reply:
@devimc : Updated my PR